conda/core/initialize.py

```diff
@@ -816,6 +849,17 @@ def install_conda_fish(target_path, conda_prefix):
     file_content = FishActivator().hook(auto_activate_base=False)
     return _install_file(target_path, file_content)
 
+def install_conda_psm1(target_path, conda_prefix):
+    # target_path: join(conda_prefix, 'shell', 'condabin', 'Conda.psm1')
+    conda_psm1_path = join(CONDA_PACKAGE_ROOT, 'shell', 'condabin', 'Conda.psm1')
+    with open(conda_psm1_path) as fsrc:
+        file_content = fsrc.read()
+    return _install_file(target_path, file_content)
+
+def install_conda_hook_ps1(target_path, conda_prefix):
+    # target_path: join(conda_prefix, 'shell', 'condabin', 'conda-hook.ps1')
+    file_content = PowerShellActivator().hook(auto_activate_base=False)
+    return _install_file(target_path, file_content)
 
```

Comment:
Since I've already added a few code style consistency comments, might as well put two newlines here and at line 858.  I'm actually surprised flake8 didn't complain at you.  I guess we must have suppressed that rule originally, but probably time to turn it back on.

Reply:
Fixed.